home/templates/home/index.html

```diff
@@ -3,10 +3,88 @@
 
 {% block content %}
 
-    <!-- TO DO - help wanted -->
-    <h1>TO DO : home > templates > home > index.html</h1>
-    <a class="btn btn-ci" href="{% url 'home' %}">Sample Button</a>
-    <br>
-    <small>custom class: <strong>btn-ci</strong></small>
+   <div class="container-fluid">
+    <div class="row">
+        <div class = "col-6 col-sm-12  col-md-6">
+            <h1 class=" text text-center">Upcoming Hackathons</h1>
+            <div id="accordion">
+            <h3>Hackathon 01 Title</h3>
+                <div>
+                    <p>
+                    Mauris mauris ante, blandit et, ultrices a, suscipit eget, quam. Integer

```

Comment:
Even if it's fake Hackathon data, something a bit more suitable to the project would be great, instead of Lorem text for example, Would you mind providing some "fake" Hackathon details. Feel free to checkout the #hackathons channel on Slack and just use the details from a previous Hackathon to get some inspiration of details (dates/times/theme/etc.).

Reply:
ok will do it now and update!